apps/ledger-live-desktop/src/renderer/analytics/segment.ts

```diff
@@ -236,6 +236,8 @@ export const updateIdentify = async () => {
     context: getContext(),
   });
 };
+/** Ensure PTX flag attributes are set as soon as feature flags load */
+runOnceWhen(() => !!analyticsFeatureFlagMethod && !!getAnalytics(), updateIdentify);
```

Comment:
I'm not sure this is even necessary since it no longer has the extra properties being added to the user traits, and the extraProperties re-run all the time. But there's not much harm to leave it in since it only runs once.